src/util/logger-utils.ts

```diff
@@ -0,0 +1,82 @@
+import { Writable } from "stream";
+import safeJsonStringify from "safe-json-stringify";
+import bformat from "bunyan-format";
+import { DEBUG } from "bunyan";
+import { createHashWithSharedSecret } from "utils/encryption";
+
+const SENSITIVE_DATA_FIELDS = ["jiraHost", "orgName", "repoName", "userGroup", "userGroup", "aaid", "username"];
+// For any Micros env we want the logs to be in JSON format.
+// Otherwise, if local development, we want human readable logs.
+const outputMode = process.env.MICROS_ENV ? "json" : "short";
+
+//TODO Remove this code when there will be convenient way to do it in Probot.
+//  See https://github.com/probot/probot/issues/1577
+export const filterHttpRequests = (record: Record<string, any>, filteredLoggerName: string) => {
+	const { msg, name } = record;
+	if (name !== filteredLoggerName) {
+		return false;
+	}
+	return !!msg.match(/(GET|POST|DELETE|PUT|PATCH)/);
+};
+
+class RawLogStream extends Writable {
+	private readonly  filteredHttpLoggerName: string;
```

Comment:
Unsafe stream: hashes nothing, but only writes out debug level or lower to a locked down logging environment